### PR TITLE
Add a connection indicator 

### LINKED ISSUE #54: Add NetworkTable connection indicator
I think this should be built into the UI (like in the title bar) and not a widget.

### PR DESCRIPTION
**Pull request by AustinShalit**
closes #54

### CODE REVIEW

**Comment by AustinShalit** [comment]
Welp... started at the wrong commit.  Going to rebase

**Comment by SamCarlberg** [comment]
I can't get this to work when connecting/disconnecting from a local server (outlineviewer). The label always says "Connected to 127.0.0.1" even when I change the remote ID to something like "190" or "8.8.8.8". This might be a problem with network tables not properly updating the server

I've already fixed this on a different branch locally. However, I'm worried about binding too closely to network tables in the app. This should belong in the NetworkTables plugin

File: app/src/main/resources/edu/wpi/first/shuffleboard/app/PlaybackControls.fxml [hunk 1]
```diff
@@ -8,44 +8,55 @@
...
+                    <ImageView>
+                        <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Fast Forward-16.png"/>
+                    </ImageView>
+                </graphic>
+            </Button>
+            <HBox alignment="CENTER" spacing="4">
+                <Scrubber fx:id="progressScrubber"/>
+                <ToggleSwitch text="Loop" fx:id="loopingSwitch" alignment="CENTER" textAlignment="CENTER"/>
+                <Label fx:id="progressLabel"/>
+            </HBox>
```
Comment (nightpool) [inline app/src/main/resources/edu/wpi/first/shuffleboard/app/PlaybackControls.fxml]: This needs to be in a separate component, putting it as part of the PlaybackIndicator doesn't make any sense architecturally.

**Comment by SamCarlberg** [comment]
@AustinShalit I think this could be better implemented as a data type + widget in the NetworkTables plugin. It'd behave similarly to the old dashboard in that regard, as well as not tie the app too strongly to NetworkTables

**Comment by sciencewhiz** [comment]
SmartDashboard originally only had a widget, but now also shows connection status in the titlebar. 

image

**Comment by SamCarlberg** [comment]
Superseded by #324

**Pull request closed without merging**